_topic_map.yml

```diff
@@ -265,6 +265,8 @@ Topics:
     File: installing-openstack-installer-custom
   - Name: Installing a cluster on OpenStack with Kuryr
     File: installing-openstack-installer-kuryr
+  - Name: Installing a cluster that supports SR-IOV compute machines on OpenStack
```

Comment:
"on OpenStack with" precedes and "on OpenStack on" follows this one.  This TOC entry uses "on OpenStack" at the end.

Obligatory sugg, in case you weren't deliberately differentiating this entry, "Installing a cluster on OpenStack with SR-IOV compute machines"

Just pointing out the diff.

Reply:
It's a distinction (if a pedantic one). This assembly really gets you to a point where you can use SR-IOV compute machines. You then really need the day 2 stuff that follows it to get going.